app/src/main/java/ar/com/wolox/android/example/model/News.kt

```diff
@@ -0,0 +1,12 @@
+package ar.com.wolox.android.example.model
+
+data class News(
+    var id: Int,
+    var userId: Int,
+    var createdAt: String,
+    var title: String,
+    var picture: String,
+    var text: String
+) {
+    // Keep it empty
```

Comment:
This part it's not really necessary, you can leave it in the last `)`. Braces can be removed